src/main/java/com/sylver/watchwithme/controller/RoomController.java

```diff
@@ -0,0 +1,76 @@
+package com.sylver.watchwithme.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Controller;
+
+import com.sylver.watchwithme.model.ClientEvent;
+import com.sylver.watchwithme.model.Command;
+
+/**
+ * This Controller will handle websockets updates regarding Sync Rooms.
+ */
+@Controller
+public class RoomController {
+
+	private static final Logger LOG = LoggerFactory.getLogger(RoomController.class);
+
+	private final SimpMessagingTemplate messagingTemplate;
+
+	private static final String SAMPLE_EVENT_CHANNEL = "/sampleeventchannel";
+	private static final String SAMPLE_COMMAND_CHANNEL = "/samplecommandchannel";
+
+	/**
+	 * The {@link Autowired} annotation tells Spring's dependency injection magic to make whatever
+	 * the constructor requires and pass it in.  Since {@link SimpMessagingTemplate} is a thing
+	 * that Spring knows about, it can do this out of the box and we don't have to tell Spring
+	 * how to instantiate the class.
+	 * @param messagingTemplate A {@link SimpMessagingTemplate} that can be used to send messages
+	 *                          over websockets channels.
+	 */
+	@Autowired
+	public RoomController(SimpMessagingTemplate messagingTemplate) {
+		this.messagingTemplate = messagingTemplate;
+	}
+
+	/**
+	 * This method will be triggered whenever a websockets message comes into the channel specified
+	 * in the {@link MessageMapping} annotation.
+	 * @param clientEvent All messages coming from the client will be {@link ClientEvent}s.
+	 */
+	@MessageMapping(SAMPLE_EVENT_CHANNEL)
+	public void handleClientEvent(ClientEvent clientEvent) {
+		LOG.info("handleClientEvent: {}", clientEvent.toString());
```

Comment:
does this insert the second param into the curly braces? Is there a benefit to doing that over just using string concat or a string builder?